### DISCUSSION TITLE
I am very doubtful that milvus cannot be deployed in standalone mode on Windows for production.

### DISCUSSION THREAD

**Discussion by luliloveshe** [milvus-io/milvus#27946] [Q&A and General discussion]
Docker desktop deployment of Milvus standalone on windows can easily crash. Milvus becomes unavailable after a few days. Sometimes restarting docker desktop fails and the machine must be restarted. I am very doubtful that milvus cannot be deployed in standalone mode on Windows for production.

**Comment by xiaofan-luan** [comment]
> Docker desktop deployment of Milvus standalone on windows can easily crash. Milvus becomes unavailable after a few days. Sometimes restarting docker desktop fails and the machine must be restarted. I am very doubtful that milvus cannot be deployed in standalone mode on Windows for production.

you are right.
Milvus is only tested on linux env.
We hight recommend you to deploy in docker on windows

**Reply by luliloveshe** [reply]
What you mean is: Milvus deployed in standalone mode using docker on Windows can be used in production environments. Can I understand it this way? In addition, it seems that you can only install docker desktop on Windows. I tried it, but as the title says, it crashes and freezes after a few days. Sometimes restarting docker desktop does not work, I can only restart the machine, which feels very unstable. This is still tens of thousands of vectors. In this case, the actual production is over millions and tens of millions of vector stores.

**Comment by yhmo** [comment]
Indeed, we don't recommend deploying it on Windows for production.
This article might be helpful for compile it on windows: https://milvus.io/blog/2021-11-19-run-milvus-2.0-on-windows.md
